contrib/broker/k8s/broker_test.go

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main_test
+
+import (
+	"testing"
+
+	_ "github.com/kubernetes-incubator/service-catalog/contrib/broker/k8s"
```

Comment:
are these lines needed? I thought the current dir is automatically picked up?
Although, this might be because you used the "main_test" package - why?  I think I'm used to the test files being in the same package as the src.